Author: Michal Čihař <nijel@debian.org>
Subject: Include both mootools and mootools-more
  This is needed for using mootools from Debian package rather than shipped one.
--- a/db_structure.php
+++ b/db_structure.php
@@ -13,6 +13,7 @@
 require_once './libraries/Table.class.php';
 
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 
 /**
  * Prepares the tables list if the user where not redirected to this script
--- a/main.php
+++ b/main.php
@@ -12,6 +12,7 @@
 define('PMA_MOORAINBOW', true);
 require_once './libraries/common.inc.php';
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 $GLOBALS['js_include'][] = 'mooRainbow/mooRainbow.js';
 $GLOBALS['js_include'][] = 'mootools-domready-rainbow.js';
 
--- a/setup/index.php
+++ b/setup/index.php
@@ -36,6 +36,7 @@
 <link href="../favicon.ico" rel="shortcut icon" type="image/x-icon" />
 <link href="styles.css" rel="stylesheet" type="text/css" />
 <script type="text/javascript" src="../js/mootools.js"></script>
+<script type="text/javascript" src="../js/mootools-more.js"></script>
 <script type="text/javascript" src="scripts.js"></script>
 </head>
 <body>
--- a/sql.php
+++ b/sql.php
@@ -16,6 +16,7 @@
 require_once './libraries/bookmark.lib.php';
 
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 
 /**
  * Defines the url to return to in case of error in a sql statement
--- a/tbl_replace.php
+++ b/tbl_replace.php
@@ -419,6 +419,7 @@
 $GLOBALS['js_include'][] = 'functions.js';
 // in case we call sql.php which needs those:
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 
 $active_page = $goto_include;
 
--- a/tbl_select.php
+++ b/tbl_select.php
@@ -20,6 +20,7 @@
 
 $GLOBALS['js_include'][] = 'tbl_change.js';
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 
 if ($GLOBALS['cfg']['PropertiesIconic'] == true) {
     $titles['Browse'] =
--- a/tbl_structure.php
+++ b/tbl_structure.php
@@ -15,6 +15,7 @@
 require_once './libraries/relation.lib.php';
 
 $GLOBALS['js_include'][] = 'mootools.js';
+$GLOBALS['js_include'][] = 'mootools-more.js';
 
 /**
  * handle multiple field commands if required
